subscription-engine/kafka-ws/app/Kafka/KafkaConsumer.js

```diff
@@ -38,6 +38,12 @@ class KafkaConsumer {
     }
   }
 
+  /**
+   * HealthChecker to be passed to the ServiceStateManager
+   *
+   * @param {*} signalReady
+   * @param {*} signalNotReady
+   */
   healthChecker(signalReady, signalNotReady) {
```

Comment:
This isn't a good name for a method. The good practice says to use a verb not a noun. Something like checkHealth would be better.